### DISCUSSION TITLE
Currently features are buggy in either fabric or fabric-api or both

### DISCUSSION THREAD

**Discussion by davidromrell** [FabricMC/fabric-api#3692] [Player support] [answered]
### Minecraft Logs

```txt
no log
```

### Description

Any custom noise or special worldgen causes issues with features.

**Comment by davidromrell** [comment]
@haykam821 read the description first.

**Comment by Linguardium** [comment]
you can reproduce this problem with _only_ fabric modloader (no mods at all) and a datapack with custom worldgen while being unable to reproduce it on the same world and minecraft version and datapack on vanilla?

**Reply by davidromrell** [reply]
I think so I will double check.

**Reply by Linguardium** [reply, marked as answer]
if so, post the datapack you are using to reproduce the issue with only fabric modloader and not vanilla when you are running no mods at all, along with the latest.log

**Reply by davidromrell** [reply]
I am guessing it was just a really odd glitch. Odd that it comes back whenever I am not testing for it though...

**Reply by Linguardium** [reply]
no worries. if you run into it again and can reproduce it with just fabric, we'll still be here (or on the discord)
